rest_specification/restful_service_specification.md

```diff
@@ -161,7 +173,7 @@ corresponding to parameter names.
 Examples:
 
 ```json
-POST /user/findByLogin
+POST /user/v1/findByLogin
```

Comment:
I think we should take the Spring Data REST approach and throw all collection based operations under `user/search`, and keep all single item operations under `/user/1`.  I think we mentioned that in the other PR.

What's the reasoning for making this a `POST`? Do we expect to send message bodies in order to query?  I'd really prefer we don't allow that, but if we have to I can be swayed.